packages/front-end/enterprise/components/Insights/MetricCorrelations/MetricCorrelationsExperimentTable.tsx

```diff
@@ -0,0 +1,331 @@
+import React, { FC, useState } from "react";
+import { FaShippingFast } from "react-icons/fa";
+import clsx from "clsx";
+import Link from "next/link";
+import { date, datetime } from "shared/dates";
+import {
+  ExperimentMetricInterface,
+  getMetricResultStatus,
+} from "shared/experiments";
+import { DifferenceType, StatsEngine } from "back-end/types/stats";
+import {
+  ExperimentWithSnapshot,
+  SnapshotMetric,
+} from "back-end/types/experiment-snapshot";
+import {
+  ExperimentDecisionFrameworkSettings,
+  ExperimentPhaseStringDates,
+  ExperimentResultsType,
+  ExperimentStatus,
+  Variation,
+} from "back-end/types/experiment";
+import { Box } from "@radix-ui/themes";
+import { useOrganizationMetricDefaults } from "@/hooks/useOrganizationMetricDefaults";
+import useConfidenceLevels from "@/hooks/useConfidenceLevels";
+import usePValueThreshold from "@/hooks/usePValueThreshold";
+import { experimentDate } from "@/pages/experiments";
+import { useSearch } from "@/services/search";
+import Tooltip from "@/components/Tooltip/Tooltip";
+import { formatNumber } from "@/services/metrics";
+import ExperimentStatusIndicator from "@/components/Experiment/TabbedPage/ExperimentStatusIndicator";
+import ChangeColumn from "@/components/Experiment/ChangeColumn";
+import Pagination from "@/components/Pagination";
+import Checkbox from "@/components/Radix/Checkbox";
+
+interface Props {
+  experimentsWithSnapshot: ExperimentWithSnapshot[];
+  metrics: ExperimentMetricInterface[];
+  bandits?: boolean;
+  numPerPage?: number;
+  differenceType?: DifferenceType;
+  excludedExperimentVariations: {
+    experimentId: string;
+    variationIndex: number;
+  }[];
+  setExcludedExperimentVariations: (
+    experimentVariations: { experimentId: string; variationIndex: number }[]
+  ) => void;
+}
+
+export interface MetricExperimentData {
+  id: string;
+  date: string;
+  name: string;
+  status: ExperimentStatus;
+  results?: ExperimentResultsType;
+  archived: boolean;
+  variations: Variation[];
+  statsEngine: StatsEngine;
+  variationIndex: number;
+  variationName: string;
+  metricResults: {
+    results: SnapshotMetric;
+    significant: boolean;
+    lift: number | undefined;
```

Comment:
keep with the other styles 
```suggestion
    lift?: number;
```